packages/api/models/granules.js

```diff
@@ -388,6 +388,74 @@ class Granule extends Manager {
     }
     return executionDescription;
   }
+
+  /**
+   * Generate and store granule records from a Cumulus message.
+   *
+   * @param {Object} cumulusMessage - Cumulus workflow message
+   * @returns {Promise}
+   */
+  async storeGranulesFromCumulusMessage(cumulusMessage) {
```

Comment:
I think this is is a bad merge? This function is no longer used in the feature branch